packages/components/src/DataList/DataList.tsx

```diff
@@ -67,6 +67,7 @@ export function DataList<T extends DataListObject>(props: DataListProps<T>) {
         filterComponent,
         layoutComponents,
         emptyStateComponents,
+        selectedItems: props.selectedItems ?? [],
         ...props,
```

Comment:
won't this get overwritten? 

![image](https://github.com/GetJobber/atlantis/assets/15986172/dc5a9821-bf1f-42b9-9dbb-09848d6cfb57)